dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -593,7 +593,7 @@ def run_shell(verbose: bool, dry_run: bool, shell_params: ShellParams) -> RunCom
         get_console().print(f"[red]Error {command_result.returncode} returned[/]")
         if verbose:
             get_console().print(command_result.stderr)
-        return command_result
+        sys.exit(1)
```

Comment:
I just missed this one, but for posterity: any reason to not pass along the exit code we receive from the command result instead of hardcoding it to `1`?

Reply:
Ah, that was a mistake! Can you PR the fix?

Reply:
Sure, I can put up a PR for it :+1:

Reply:
@ephraimbuddy 
https://github.com/apache/airflow/pull/27048